packages/lesswrong/components/posts/PostsItem2.tsx

```diff
@@ -455,7 +496,9 @@ const PostsItem2 = ({
                       sticky={isSticky(post, terms)}
                       showQuestionTag={showQuestionTag}
                       showDraftTag={showDraftTag}
+                      showPersonalIcon={showPersonalIcon}
```

Comment:
My cursory read of this suggests that this changes the default behavior? If showPersonalIcon is not specified by the caller of this component, it will pass through `showPersonalIcon={undefined}`, which (I think) will overwrite the default `showPersonalIcon=true` of PostsTitle. You could fix that with the following:
```suggestion
                      ...{showPersonalIcon ? {showPersonalIcon} : {}}
```